Phaethon/WebClient/Controllers/HomeController.cs

```diff
@@ -3,6 +3,7 @@
 using System.Linq;
 using System.Web;
 using System.Web.Mvc;
+using a = WebClient.Resources.Language_Files.LanguagePack;
```

Comment:
realy bad naming, make it something that makes sense instead of a

Reply:
Ok